pkgs/top-level/all-packages.nix

```diff
@@ -26550,11 +26550,9 @@ with pkgs;
     glusterfsSupport = lib.meta.availableOn stdenv.hostPlatform glusterfs && lib.meta.availableOn stdenv.hostPlatform libuuid;
   });
 
-  # See `xenPackages` source for explanations.
   # Building with `xen` instead of `xen-slim` is possible, but makes no sense.
```

Comment:
Actually, it allows for much slimmer system closure, as you will no longer have two xen.efi binaries and command line utilities

But I think it is better to just use slim version everywhere by default.